### PR TITLE
Adjust the worker SSH rule more precisely to avoid side-effects

### PR DESCRIPTION
**Pull request by tuommaki**

### CODE REVIEW

File: service/controller/resource/deployment/template/main.json [hunk 1]
```diff
@@ -428,7 +428,7 @@
                       "sourcePortRange":"*",
                       "destinationPortRange":"22",
                       "sourceAddressPrefix":"[parameters('hostClusterCidr')]",
-                      "destinationAddressPrefix":"[parameters('workerSubnetCidr')]",
+                      "destinationAddressPrefix":"[parameters('virtualNetworkCidr')]",
```
Comment (whites11) [inline service/controller/resource/deployment/template/main.json:431]: You changed a .json file but I don't see the pkged.go file. Did you run pkger?

Comment (whites11) [reply service/controller/resource/deployment/template/main.json:431]: Plus it doesn't seem this parameter is defined in the ARM template

Comment (tuommaki) [reply service/controller/resource/deployment/template/main.json:431]: :scream: 

Totally forgot the whole thing.

Comment (tuommaki) [reply service/controller/resource/deployment/template/main.json:431]: Done. Tested. W0krs. Reverted `project.go`.

**Comment by tuommaki** [comment]
Going with a ping :bell:

**Pull request merged**